build/sign.proj

```diff
@@ -15,15 +15,11 @@
 
   <UsingTask TaskName="ESRPSignTask" AssemblyFile="$(CISignRepo)\tools\sdkbuildtools\tasks\MS.Az.Sdk.OnPremise.Build.Tasks.dll" />
 
-  <Target Name="netstandard20">
+  <Target Name="netstandard20" Condition="'$(DisableSigning)' != 'true'">
```

Comment:
You can remove the `DisableSigning` logic. That only existed to do the batch processing.